File: app/src/main/java/com/example/kouki/myfirstapplication/Book.java
package com.example.kouki.myfirstapplication;

import android.provider.BaseColumns;

public class Book implements BaseColumns {
    public static final String BOOK_TABLE_NAME = "book";

    public static final String COLUMN_NAME_BOOK_TITLE = "title";
    public static final String COLUMN_NAME_BOOK_PUBLISHER = "publisher";
    public static final String COLUMN_NAME_BOOK_PRICE = "price";

    private String mTitle;
    private String mPublisher;
    private int mPrice;
}

File: app/src/main/java/com/example/kouki/myfirstapplication/Book.kt
package com.example.kouki.myfirstapplication

import android.provider.BaseColumns

class Book : BaseColumns {

    private val mTitle: String? = null
    private val mPublisher: String? = null
    private val mPrice: Int = 0

    companion object {
        val BOOK_TABLE_NAME = "book"

        val _ID = "_id"
        val COLUMN_NAME_BOOK_TITLE = "title"
        val COLUMN_NAME_BOOK_PUBLISHER = "publisher"
        val COLUMN_NAME_BOOK_PRICE = "price"
    }
}
